make/sql.mk

```diff
@@ -0,0 +1,44 @@
+# This is the default Clever SQL Makefile.
+# It is stored in the dev-handbook repo, github.com/Clever/dev-handbook
+# Please do not alter this file directly.
+SQL_MK_VERSION := 0.0.1
+
+# USAGE:
+# - safesql
+#   In your project's Makefile, run safesql as part of the `test` command. Example:
+# 	```
+# 	test: $(PKGS) safesql
+# 	```
+# 	In test output, you will see "SCANNING SQL" and then the output of the safesql tool.
+
+
+SHELL := /bin/bash
+.PHONY: run-safesql run-safesql-deps
+
+# if the gopath includes several directories, use only the first
+GOPATH=$(shell echo $$GOPATH | cut -d: -f1)
+
+# safesql is a static analysis tool for Go that checks for vulnerability to SQL injections.
+safesql: run-safesql-deps
+	$(call run-safesql,$(PKG))
+
+# to install and runs safesql
+SAFESQL := $(GOPATH)/bin/safesql
+$(SAFESQL):
+	go get github.com/stripe/safesql
+
+# run-safesql-deps requires the SAFESQL tool
+run-safesql-deps: $(SAFESQL)
+
+# run-safesql runs safesql on the pkg.
```

Comment:
How are going to deal with cases where safesql needs to "fail"? Is there an exemption process in code? Would love to understand how a developer will interact with this code.

Reply:
Added mentions of safesql's line-by-line ignore.